.github/workflows/cwag-workflow.yml

```diff
@@ -0,0 +1,272 @@
+        if: success() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL_ARTIFACTS }}
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_OSS }}
+          SLACK_TITLE: "*Cwag Artifact Has Been Published*"
+          SLACK_USERNAME: "Cwag workflow"
+          SLACK_ICON_EMOJI: ":heavy_check_mark:"
+          SLACK_COLOR: "#00FF00"
+  xwfm-deploy-latest:
+    needs:
+      - cwag-deploy
+      - xwfm-test
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      DOCKER_REGISTRY: "${{ secrets.DOCKER_REGISTRY }}"
+      DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
+      DOCKER_PASSWORD: "${{ secrets.DOCKER_PASSWORD }}"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build xwf go radius
+        run: |
+              cd ${MAGMA_ROOT}/feg
+              docker build --build-arg BUILD_NUM=${CIRCLE_SHA1:0:8} --tag goradius -f radius/src/Dockerfile ./
+      - name: Tag and push to Docker Registry
+        if: github.ref == 'refs/heads/master'
+        # yamllint disable rule:line-length
+        run: |
+            ./ci-scripts/tag-push-docker.sh --images 'goradius' --tag '${GITHUB_SHA:0:8}' --tag-latest true --project cwag
+      - name: Load openvswitch kernel module for xwf integ test
+        # yamllint enable
+        run: sudo modprobe openvswitch
+      - name: Build xwfm-integ-tests
+        run: |
+              cd ${MAGMA_ROOT}
+              docker build --tag xwfm-integ-tests -f xwf/gateway/integ_tests/gw/Dockerfile ./
+      - name: Tag and push to Docker Registry
+        if: github.ref == 'refs/heads/master'
+        # yamllint disable rule:line-length
+        run: |
+            ./ci-scripts/tag-push-docker.sh --images 'xwfm-integ-tests' --tag "${GITHUB_SHA:0:8}" --tag-latest true --project cwag
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      # yamllint enable
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action xwfm-deploy-latest failed"
+          SLACK_USERNAME: "Cwag workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+      # Notify ci channel when push succeeds
+      - name: Notify success to slack
+        if: success() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_CHANNEL: ${{ secrets.SLACK_CHANNEL_ARTIFACTS }}
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_OSS }}
+          SLACK_TITLE: "*xwfm Artifact Has Been Published*"
+          SLACK_USERNAME: "Cwag workflow"
+          SLACK_ICON_EMOJI: ":heavy_check_mark:"
+          SLACK_COLOR: "#00FF00"
+  cwag-deploy-latest:
```

Comment:
looks like this job is useless, it do the same thing as cwag-deploy but with less images.
I see it in circle ci but let's delete it

Reply:
sure!